cmd/geth/chaincmd.go

```diff
@@ -411,12 +409,12 @@ func copyDb(ctx *cli.Context) error {
 	fmt.Printf("Database copy done in %v\n", time.Since(start))
 
 	// Compact the entire database to remove any sync overhead
-	start = time.Now()
+	/*start = time.Now()
```

Comment:
So explicit compaction is now gone?

Reply:
Ah, yes. Compaction requires casting it down to `leveldb.Database`. I could add compactions to the database interface.

Reply:
We have this:
```
	NoCompactionFlag = cli.BoolFlag{
		Name:  "nocompaction",
		Usage: "Disables db compaction after import",
	}
```
I make use of it in hive to prevent compaction. If you have disabled explicit compaction after import anyway, I guess you could remove that flag? I didn't actually see any change in that area, but I assume you must have done _something_ on that part too?  

But perhaps it would be better to add it to the interface.

Reply:
Please see my last commit. I've added compaction support back.